package.json

```diff
@@ -79,6 +80,7 @@
     "eslint-plugin-jsx-a11y": "^4.0.0",
     "eslint-plugin-react": "^6.10.0",
     "jest": "^19.0.2",
+    "ncp": "^2.0.0",
```

Comment:
This package is used because it has a ```CLI``` option which works both on windows and mac